merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetSimulationTopicsAction.java

```diff
@@ -35,25 +35,25 @@ public List<Triple<Integer, String, ValueSchema>> get(
       final long datasetId) throws SQLException
   {
     this.statement.setLong(1, datasetId);
-    final var resultSet = this.statement.executeQuery();
-
-    final var topics = new ArrayList<Triple<Integer, String, ValueSchema>>();
-    while (resultSet.next()) {
-      topics.add(
-          Triple.of(
-              resultSet.getInt(1),
-              resultSet.getString(2),
-              parseValueSchema(resultSet.getString(3))
-          )
-      );
+    try (final var resultSet = this.statement.executeQuery()) {
+      final var topics = new ArrayList<Triple<Integer, String, ValueSchema>>();
+      while (resultSet.next()) {
+        topics.add(
+            Triple.of(
+                resultSet.getInt(1),
+                resultSet.getString(2),
+                parseValueSchema(resultSet.getString(3))
```

Comment:
This action doesn't follow the pattern established by all of the other actions, which use `getCharacterStream` instead of `getString`. I know it's not the fault of this PR, but I'd recommend changing this to use `getCharacterStream` in line with everything else.

Reply:
Made redundant by refactoring with the new getJsonColumn method you suggested.

Reply:
🚀